Tianheng_HW1/css/stylesheet.css

```diff
@@ -0,0 +1,49 @@
+*{
+    font-family: Verdana, Geneva, sans-serif;
+}
+header{
+    font-size: 50px;
+    text-align: center;
+    margin-bottom: 20px;
+    margin-top: 50px;
+}
+h1{
+    color: #f07c17;
+}
+ul{
+    color: #666666;
+    font-size: 13px;
+    line-height:25px;
+
+}
+main{
+    float: left;
+    width:55%;;
+    line-height: 30px;
+    margin-left: auto;
+    margin-right: auto;
+}
+aside{
+    float: left;
+    width:43%;
+    margin-left: auto;
+    margin-right: auto;
+}
+#MainFeature{
```

Comment:
usually, by convention, we just use class for styles purpose and use id for javascript to hook up element